resteasy-jaxrs/src/main/java/org/jboss/resteasy/util/Base64.java

```diff
@@ -1388,7 +1388,7 @@ else if (len < 4)
             }   // end try
             catch (java.io.IOException e)
             {
-               e.printStackTrace();
+               Messages.MESSAGES.unableToDecodeGZIPBase64();
```

Comment:
The i18n message here should be printed in the log, shouldn't it?
Moreover, I'd also like the IOException to be logged too, to simplify troubleshooting.